suite-common/fiat-services/src/rates.ts

```diff
@@ -82,6 +82,10 @@ export const fetchCurrentFiatRates = async ({
 
     const coingeckoResponse = await coingeckoService.fetchCurrentFiatRates(ticker);
 
+    if (!coingeckoResponse) {
+        return null;
+    }
+
```

Comment:
It would be good to add this also to `fetchLastWeekFiatRates` and `getFiatRatesForTimestamps` functions

Reply:
1e868530ef7b26a641d67d257f6b07531de5faf9